staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -216,6 +219,14 @@ enum NamespaceMode {
     // For example, a container with a PID namespace of NODE expects to view
     // all of the processes on the host running the kubelet.
     NODE      = 2;
+    // A NODE_WIDE_REMAPPED namespace applies to all pods on a given kubernetes node.
+    // The uid/gids of the pods/containers on a given node are mapped to a range of uids/gids
+    // from that Node's namespace. For e.g. starting with 200000 id on the nodes namespace,
+    // 10000 ids are allocated for a pod. This means uid/gid 0 inside the pod would map to 200000 id on the nodes namespace.
+    // For example, starting from uid/gid 0, 10000 uids/gids in this namespace are mapped to
+    // 10000 ids on node namespace,starting with id 200000. i.e uid/gid 0 in this namespace is
+    // mapped to uid/gid 200000 on node namespace.
```

Comment:
```suggestion
    // A NODE_WIDE_REMAPPED namespace applies to all pods on a given kubernetes node.
    // The uid/gids of the pods/containers on a given node are mapped to a range of uids/gids
    // on the root user namespace of that node. For e.g. starting with 200000 id on the nodes namespace,
    // 10000 ids are allocated for a pod. This means uid/gid 0 inside the pod would map to 200000 id on the node.
```

Or something similar...

Reply:
I think "from that Node's namespace" was better than "on the root user namespace of that node". There might be Kubernetes setups where a Kubernetes Node runs inside a user namespace.

I know that the the kubernetes administrator certification program runs Kubernetes nodes in LXC, but I am not sure if LXC is configured to use user namespaces for the worker nodes.